playbooks/common/private/components.yml

```diff
@@ -51,6 +51,9 @@
 - import_playbook: ../../openshift-service-catalog/private/config.yml
   when: openshift_enable_service_catalog | default(true) | bool
 
+- import_playbook: ../../olm/private/config.yml
+  when: openshift_enable_olm | default(true) | bool
```

Comment:
Default here should be false if we want this to be opt-in. Once we want it enabled by default revert it back to true.

Reply:
Let me check if this should be opt-in or default.

Reply:
Opt-in, so I'll change this